packages/website/src/bin/handle-config.test.js

```diff
@@ -9,18 +9,29 @@ describe('website configuration processor', () => {
 
     expect(config).toEqual({
       packagesPaths: ['/c/w/d/packages/*'],
-      docsPath: '/c/w/d/docs',
+      docsList: {
+        description: 'This is a sample documentation for the website.',
+        docsPath: '/c/w/d/docs',
+        name: 'Docs',
+      },
       useManifests: false,
       siteName: 'Brisk Docs',
       webpack: expect.any(Function),
       showSubExamples: false,
+      packagesDescription:
+        'This is a sample package for the documentation website',
     });
   });
 
   it('accepts a docs path as a string', () => {
-    const config = processConfig(mockCwd, { docs: 'some/other/docs' });
-
-    expect(config.docsPath).toEqual('/c/w/d/some/other/docs');
+    const { docsList } = processConfig(mockCwd, {
+      docs: {
+        path: 'some/other/docs',
+        name: 'Docs',
+        description: 'This is a sample documentation for the website.',
```

Comment:
Are we testing the description and packageDescription? Can we?

Also, for test data, it's good to not use just the default values, to make sure that the values are being treated.